src/xdocs/sun_style.xml

```diff
@@ -366,8 +366,23 @@
                     5.1.1 Block Comments
                   </a>
                 </td>
-                <td>???</td>
-                <td/>
+                <td>
+                  <span class="wrapper inline">
+                    <img src="images/ok_blue.png" alt=""/>
+                  </span>
+                  <a href="config_misc.html#CommentsIndentation">
+                    CommentsIndentation
```

Comment:
We need to put comment here that we do not cover
>```
block comment should be preceded by a blank line to set it apart from the rest of the code.

/*
 * Here is a block comment.
 */
```
It should be done by some other Chek

Reply:
We do not have check that demand empty line in block comment :)